MELA/interface/Mela.h

```diff
@@ -345,7 +352,10 @@ class Mela{
   double selfDaTQGCcoupl[SIZE_ATQGC][2];
   //****Anomnalous Zff**//
   double selfDAZffcoupl[SIZE_AZff][2];
-  // That is a lot of them!
+  //****SMEFTSIM terms**//
+  double selfDSmeftSimcoupl[SIZE_SMEFT]; //madMela
```

Comment:
selfDSmeftSimcoupl stores the SMEFTsim couplings through `TUtil::SetMadgraphSpinZeroCouplings` and the `CouplingIndex_Smeft` enum in `TCouplingsBase.hh`